core/docker/importer.sh

```diff
@@ -1,2 +1,23 @@
+#!/bin/sh
 
-curl -X POST --insecure -u "$OPENHIM_API_USERNAME:$OPENHIM_API_PASSWORD" -H "Content-Type: application/json" -d @openhim-import.json "$OPENHIM_API_SERVER/metadata" -v
\ No newline at end of file
+OPENHIM_RESPONSE="";
+TIMEOUT=0
+
+while [ "$OPENHIM_RESPONSE" != "200" ]
+do
+  TIMEOUT=$((TIMEOUT+1))
+  # Fail script if incomplete at 30 seconds
+  if [ "$TIMEOUT" == "15" ]
+  then
+    echo -e "Sleep timeout reached: $TIMEOUT.\nFailed to import config. Could not reach the OpenHIM."
+    exit 1
+  fi
+
+  echo "OpenHIM not ready ( $OPENHIM_RESPONSE ) - sleeping"
+  sleep 2
+  OPENHIM_RESPONSE=$(curl -X GET --insecure --write-out %{http_code} --silent --output /dev/null "$OPENHIM_API_SERVER/heartbeat");
+done
+
+echo -e "\nSTART Posting OpenHIM Config\n----------------------------\n"
+curl --insecure -u "$OPENHIM_API_USERNAME:$OPENHIM_API_PASSWORD" -H "Content-Type: application/json" -d @openhim-import.json "$OPENHIM_API_SERVER/metadata" -v
+echo -e "\n\n--------------------------\nEND Posting OpenHIM Config\n"
```

Comment:
Maybe we need to interpret the response here to inform the user of the status. If the json payload isnt formatted correctly, the final response will still print out that the OpenHIM config has been imported